xcodeproject/testhelper/testhelper.go

```diff
@@ -34,12 +34,17 @@ func GitCloneBranchIntoTmpDir(t *testing.T, repo string, branch string) string {
 	tmpDir, err := pathutil.NormalizedOSTempDirPath("__xcode-proj__")
 	require.NoError(t, err)
 
-	cmd := command.New("git", "clone", "-b", branch, repo, tmpDir)
-	require.NoError(t, cmd.Run())
+	GitCloneBranch(t, repo, branch, tmpDir)
 
 	return tmpDir
 }
 
+// GitCloneBranch clones a branch from a git repository into an existing directory
+func GitCloneBranch(t *testing.T, repo string, branch string, dir string) {
+	cmd := command.New("git", "clone", "-b", branch, repo, dir)
```

Comment:
nit: it's worth adding `--depth=1` for fast tests

Reply:
added in: https://github.com/bitrise-io/go-xcode/pull/207/commits/473d0c3d31c62888580918cadf759e5428f684a4